recipes/deeptools/meta.yaml

```diff
@@ -0,0 +1,45 @@
+package:
+  name: deeptools
+  version: '2.0.0'
+
+source:
+  fn: deepTools-2.0.0.tar.gz
+  url: https://pypi.python.org/packages/source/d/deepTools/deepTools-2.0.0.tar.gz
+  md5: 09821e73f56ddb744a35d01173c6eb29
+
+build:
+  number: 0
+  skip: True # [not py27]
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - numpy x.x
+    - scipy >=0.15.0
+    - matplotlib >=1.4.0
+    - pysam >=0.8.2
+    - bx-python >=0.7.2
+  run:
+    - python
+    - pybigwig >=0.2.3
+    - numpy x.x
+    - scipy >=0.15.0
+    - matplotlib >=1.4.0
+    - pysam >=0.8.2
+    - bx-python >=0.7.2
+
+test:
+  imports:
+    - deeptools
+
+  commands:
+    - nosetests --with-doctest deeptools
```

Comment:
The nosetests are not needed here. They will only slow down our build process. The purpose of the test section is only to test if a proper package was generated. Not if the release itself is free of bugs.